### PR TITLE
[SCB-1304]FallbackPolicy#getFallbackResponse add Throwable

### PR DESCRIPTION
**Pull request by wuliuqi666**
Follow this checklist to help us incorporate your contribution quickly and easily:

 - [ ] Make sure there is a [JIRA issue](https://issues.apache.org/jira/browse/SCB) filed for the change (usually before you start working on it).  Trivial changes like typos do not require a JIRA issue.  Your pull request should address just this issue, without pulling in other changes.
 - [ ] Each commit in the pull request should have a meaningful subject line and body.
 - [ ] Format the pull request title like `[SCB-XXX] Fixes bug in ApproximateQuantiles`, where you replace `SCB-XXX` with the appropriate JIRA issue.
 - [ ] Write a pull request description that is detailed enough to understand what the pull request does, how, and why.
 - [ ] Run `mvn clean install -Pit` to make sure basic checks pass. A more thorough check will be performed on your pull request automatically.
 - [ ] If this contribution is large, please file an Apache [Individual Contributor License Agreement](https://www.apache.org/licenses/icla.pdf).

---

### CODE REVIEW

File: handlers/handler-bizkeeper/src/main/java/org/apache/servicecomb/bizkeeper/ThrowExceptionFallbackPolicy.java [hunk 1]
```diff
@@ -30,7 +30,7 @@ public String name() {
   }
 
   @Override
-  public Response getFallbackResponse(Invocation invocation) {
+  public Response getFallbackResponse(Invocation invocation, Throwable error) {
```
Comment (wujimin) [inline handlers/handler-bizkeeper/src/main/java/org/apache/servicecomb/bizkeeper/ThrowExceptionFallbackPolicy.java]: need change second parameter from null to error of createBizkeeperException, or not?

File: handlers/handler-bizkeeper/src/main/java/org/apache/servicecomb/bizkeeper/FallbackPolicy.java [hunk 2]
```diff
@@ -22,7 +22,7 @@
 public interface FallbackPolicy {
   String name();
 
-  Response getFallbackResponse(Invocation invocation);
+  Response getFallbackResponse(Invocation invocation, Throwable error);
```
Comment (wujimin) [inline handlers/handler-bizkeeper/src/main/java/org/apache/servicecomb/bizkeeper/FallbackPolicy.java]: seems not compatible?

Comment (liubao68) [reply handlers/handler-bizkeeper/src/main/java/org/apache/servicecomb/bizkeeper/FallbackPolicy.java]: I think the change is fine. Users can change their implementations easily.

File: handlers/handler-bizkeeper/src/main/java/org/apache/servicecomb/bizkeeper/ThrowExceptionFallbackPolicy.java (hunk 1 above)
Comment (liubao68) [reply handlers/handler-bizkeeper/src/main/java/org/apache/servicecomb/bizkeeper/ThrowExceptionFallbackPolicy.java]: It's better to use error as the parameter, I think

File: handlers/handler-bizkeeper/src/test/java/org/apache/servicecomb/bizkeeper/TestFallbackPolicyManager.java [hunk 3]
```diff
@@ -33,6 +33,36 @@ public void testFallbackPolicyManager(final @Mocked Configuration config, final
...
+      private static final String CUSTOM = "custom";
+
+      @Override
+      public String name() {
+        return CUSTOM;
+      }
+
+      @Override
+      public Response getFallbackResponse(Invocation invocation, Throwable error) {
+        return Response.succResp("test");
```
Comment (liubao68) [inline handlers/handler-bizkeeper/src/test/java/org/apache/servicecomb/bizkeeper/TestFallbackPolicyManager.java]: This test case is better to include some message from error and check it.

**Pull request merged**